modules/sandboxed-containers-deleting-operator-deployment-cr.adoc

```diff
@@ -0,0 +1,38 @@
+//Module included in the following assemblies:
+//
+// *uninstalling-sandboxed-containers.adoc
+
+[id="sandboxed-containers-deleting-operator-deployment-cr_{context}"]
+= Deleting the {sandboxed-containers-operator} deployment custom resource
+
+To uninstall {sandboxed-containers-first}, you must first delete the {sandboxed-containers-first} custom resource `KataConfig`. This removes and uninstalls the `kata` runtime and its related resources from your cluster.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed on your cluster.
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. From the {product-title} web console, select `openshift-sandboxed-containers` from the *Projects* list.
+. Navigate to the *Operators* → *Installed Operators* page.
+. Click *{sandboxed-containers-first}*.
+. Click the *{sandboxed-containers-operator}* tab.
+. Click the scroll-down list in the *Operator Details*, and click *Delete `KataConfig`*.
+. Click *Delete* in the confirmation window.
+
+Alternatively, you can use the OpenShift CLI (`oc`) to retrieve all pods that use `kata` as `runtimeClassName` by running the following command:
```

Comment:
This is not alternative, rather this is a different step. 

The first step is delete the KataConfig from the console, the second missing step is delete the pods and deployments,... again from the console and for that second step an alternative would be to delete using the CLI.